plugins/woocommerce/tests/e2e-pw/tests/merchant/create-page.spec.js

```diff
@@ -0,0 +1,73 @@
+const { test, expect, request } = require( '@playwright/test' );
+const { admin } = require( '../../test-data/data' );
+
+const pageTitle = `Page-${ new Date().getTime().toString() }`;
+
+test.describe( 'Can create a new page', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.afterAll( async ( { baseURL } ) => {
+		const base64auth = Buffer.from(
+			`${ admin.username }:${ admin.password }`
+		).toString( 'base64' );
+		const wpApi = await request.newContext( {
+			baseURL: `${ baseURL }/wp-json/wp/v2/`,
+			extraHTTPHeaders: {
+				Authorization: `Basic ${ base64auth }`,
+			},
+		} );
+
+		let response = await wpApi.get( `pages` );
+		const allPages = await response.json();
+
+		await allPages.forEach( async ( page ) => {
+			if ( page.title.rendered === pageTitle ) {
+				response = await wpApi.delete( `pages/${ page.id }`, {
+					data: {
+						force: true,
+					},
+				} );
+			}
+		} );
+	} );
+
+	test( 'can create new page', async ( { page } ) => {
+		await page.goto( 'wp-admin/post-new.php?post_type=page' );
+
+		const welcomeModalVisible = await page
+			.getByRole( 'heading', {
+				name: 'Welcome to the block editor',
+			} )
+			.isVisible();
+
+		if ( welcomeModalVisible ) {
+			await page.getByRole( 'button', { name: 'Close' } ).click();
```

Comment:
I had do do a minor tweak and update this selector, because it was updated in WC.  Aside from that, all good!

Reply:
Thank you, @lanej0 🙇